extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/linkHandler/YoutubeSearchQueryHandlerFactory.java

```diff
@@ -1,34 +1,23 @@
 package org.schabi.newpipe.extractor.services.youtube.linkHandler;
 
-import static org.schabi.newpipe.extractor.utils.Utils.encodeUrlUtf8;
-import static org.schabi.newpipe.extractor.utils.Utils.isNullOrEmpty;
-
 import org.schabi.newpipe.extractor.exceptions.ParsingException;
 import org.schabi.newpipe.extractor.linkhandler.SearchQueryHandlerFactory;
+import org.schabi.newpipe.extractor.search.filter.FilterItem;
+import org.schabi.newpipe.extractor.services.youtube.search.filter.YoutubeFilters;
 
-import java.io.UnsupportedEncodingException;
 import java.util.List;
 
 import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
 
 public final class YoutubeSearchQueryHandlerFactory extends SearchQueryHandlerFactory {
 
     private static final YoutubeSearchQueryHandlerFactory INSTANCE =
             new YoutubeSearchQueryHandlerFactory();
 
-    public static final String ALL = "all";
-    public static final String VIDEOS = "videos";
-    public static final String CHANNELS = "channels";
-    public static final String PLAYLISTS = "playlists";
-
-    public static final String MUSIC_SONGS = "music_songs";
-    public static final String MUSIC_VIDEOS = "music_videos";
-    public static final String MUSIC_ALBUMS = "music_albums";
-    public static final String MUSIC_PLAYLISTS = "music_playlists";
-    public static final String MUSIC_ARTISTS = "music_artists";
-
-    private static final String SEARCH_URL = "https://www.youtube.com/results?search_query=";
-    private static final String MUSIC_SEARCH_URL = "https://music.youtube.com/search?q=";
+    private YoutubeSearchQueryHandlerFactory() {
+        super(new YoutubeFilters());
+    }
 
     @Nonnull
     public static YoutubeSearchQueryHandlerFactory getInstance() {
```

Comment:
I saw that the creation of link handler factory instances was being done with a lazy `getInstance`, before the rebase. Since in the meantime we also added `getInstance` in the current extractor, I kept the current version to reduce changes, and thus discarded the lazy initialization. However we might want to check if it's better to lazily initialize all link handler factories to reduce static initialization time (but this can be done in a separate PR).